skymp5-server/cpp/mp_common/Networking.cpp

```diff
@@ -79,7 +79,7 @@ class Client : public Networking::IClient
   void Send(Networking::PacketData data, size_t length, bool reliable) override
   {
     peer->Send(reinterpret_cast<const char*>(data), length, MEDIUM_PRIORITY,
-               reliable ? RELIABLE : UNRELIABLE, 0, serverGuid, false);
+               reliable ? RELIABLE_ORDERED : UNRELIABLE, 0, serverGuid, false);
```

Comment:
I think it would be better to support multiple values here to allow client/server code to use the desired one. I think that it does hit performance (didn't check RakNet/SLikeNet docs though)